### ISSUE TITLE
What is the function of your wind sensor?

### ISSUE THREAD

**Issue by 369475274** [callum-rhodes/gabp_mapping#5]
I didn't find any data from the wind speed sensor in your data package,but In your program, I saw the program of the anemometer, but it doesn't seem to have specific functions.

**Comment by callum-rhodes** [comment]
Yes, that is a placeholder for some development work I was doing at the time to include modelling the wind field and gas distribution jointly using Gongora et. al's "Joint estimation of gas and wind maps for fast-response applications". Since modelling wind wasn't a focus of the paper I never finished the development version but it shouldn't be too difficult to include.
